PowerShell/ScubaGear/Testing/Unit/PowerShell/ScubaConfig/ScubaConfigJsonDuplicateProviderTests.ps1

```diff
@@ -0,0 +1,20 @@
+using module '..\..\..\..\Modules\ScubaConfig\ScubaConfig.psm1'
+
+InModuleScope ScubaConfig {
+    Describe -tag "Utils" -name 'ScubaConfigJson' {
+        context 'JSON Configuration' {
+            BeforeEach{
+                [ScubaConfig]::ResetInstance()
+                $Config = [ScubaConfig]::GetInstance()
+		$Config.Configuration.ProductNames = @('aad', 'aad')
+            }
+            It 'Should not contain duplicate product names'{
+                $Config.Configuration.ProductNames | Should -BeExactly @('aad')
+            }
+	    AfterAll {
+              [ScubaConfig]::ResetInstance()
```

Comment:
I saw the Issue on ResetInstance  I modified locally ClearConfiguration to just say 
```
$this.Configuration = $null
```

Ithink we should fix both here since hard to test your changes with that not working.  Of cource this breaks the ScubaConfigDelete.Tests.ps1 that would need to be reworked

Reply:
Switched it to use ResetInstance instead of $null to avoid issues